api/src/bigquerytest/resources/bigquery/cbdata/ds_survey_data.json

```diff
@@ -7,5 +7,14 @@
     "QUESTION": "Question",
     "ANSWER_CONCEPT_ID": 1,
     "ANSWER": "NO"
+  },
+  {
+    "PERSON_ID": "1",
+    "SURVEY_DATETIME": "2020-01-31 00:00:00 UTC",
+    "SURVEY": "The Basics",
```

Comment:
Should this survey name be: "Personal and Family Health History" ?

Reply:
Done